client/lib/products-values/translations.js

```diff
@@ -111,15 +113,14 @@ export const getJetpackProductsCallToAction = () => {
 			} ) }
 		</>
 	);
-	const search = isEnabled( 'plans/alternate-selector' )
-		? translate( 'Get Jetpack Search' )
-		: translate( 'Get Search' );
-	const scan = isEnabled( 'plans/alternate-selector' )
-		? translate( 'Get Jetpack Scan' )
-		: translate( 'Get Scan' );
-	const antiSpam = isEnabled( 'plans/alternate-selector' )
-		? translate( 'Get Jetpack Anti-spam' )
-		: translate( 'Get Anti-spam' );
+	const search =
+		currentCROvariant === 'v1' ? translate( 'Get Jetpack Search' ) : translate( 'Get Search' );
```

Comment:
I wonder if it would be helpful to have a function for `isAlternateSelector` to always include v1 and v2; or, if it would make things easier to reverse the conditions so that we're checking against v0 instead.

Reply:
I wondered the same while I was creating this PR and I didn't only because it isn't clear to me how different will v2 be in comparison to v1.  In some cases, v2 uses copy from v0, so is not always v0 versus v1 and v2. 

@mattgawarecki, would it be fine, in your opinion, if we left this PR as it is, and I add that utility function in a follow-up PR that I have almost ready for review?

Reply:
By all means!